src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTask.java

```diff
@@ -409,6 +416,7 @@ protected void closeResources() {
           }
         } else {
           consecutiveEmptyResults.put(querier, 0);
+          retriesAttemptedPerTableQuerier.put(querier, 0);
```

Comment:
Why are we resetting only if there are results? Even no results is a success, right?